util/conan-package/conanfile.py

```diff
@@ -0,0 +1,20 @@
+import os
+from conans import ConanFile
+
+class GlmConan(ConanFile):
+    name = "glm"
+    version = "master"
+    generators = "txt"
+    url="https://github.com/g-truc/glm"
+    description="OpenGL Mathematics (GLM)"
+    license = "https://github.com/g-truc/glm/blob/manual/copying.txt"
+    exports = ["FindGLM.cmake", "lib_licenses/*", os.sep.join([".", "..", "..", "*"])]
```

Comment:
You might take advantage of the ``exports_sources`` functionality to store the sources and the .cmake file. I would only keep the license as ``exports``. It is much more efficient to use ``exports_sources`` for source code.